app/controllers/api/v1/podcasts_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Api::V1::PodcastsController < ApplicationController
+  protect_from_forgery unless: -> { request.format.json? }
+
+  def index
+    render json: Podcast.all, adapter: :json
+  end
+
+  def show
+    render json: Podcast.where(id: params[:id]), adapter: :json
+  end
```

Comment:
Per this reading, we can have you avoid the `adapter: :json` designation by changing one of your config files
https://learn.launchacademy.com/teams/boston-23/curricula/on-campus-boston-23/lesson_groups/week_6:_rails/lessons/building-an-api-in-rails

I might allow this for this PR if you promise to change this in the future